### PR TITLE
Update kops_create_secret_dockerconfig.md

### PR DESCRIPTION
**Pull request by integrii**
Today we were implementing an authenticated docker user, but it was unclear how exactly to do that.  We learned that simply making this secret within kops was all that was needed for the docker config to start appearing on newly built nodes.  It would be nice if the documentation here reflected that.  It would have saved us some time.

### CODE REVIEW

File: cmd/kops/create_secret_dockerconfig.go [hunk 1]
```diff
@@ -35,7 +35,12 @@ var (
 	createSecretDockerconfigLong = templates.LongDesc(i18n.T(`
 	Create a new docker config, and store it in the state store.
 	Used to configure docker on each master or node (i.e. for auth)
-	Use update to modify it, this command will only create a new entry.`))
+	Use update to modify it, this command will only create a new entry.
+	
+	After creating a dockerconfig secret, a .docker/config.json file will 
+	be added to all newly created nodes without running kops update.
+	This file should also apply auth to containerd, which is the default
+	container runtime in kops 1.20 and above.`))
```
Comment (hakman) [inline cmd/kops/create_secret_dockerconfig.go:43]:
```suggestion
	After creating a dockerconfig secret, a /root/.docker/config.json file will 
	be added to newly created nodes (without running kops update).
	This should also work when using containerd, which is the default
	container runtime in kOps 1.20 and above.`))
```

**Review by hakman — CHANGES_REQUESTED** [review]
Thanks @integrii. Just a small suggestion from me. Please also squash the commits.

File: cmd/kops/create_secret_dockerconfig.go (hunk 1 above)
Comment (olemarkus) [reply cmd/kops/create_secret_dockerconfig.go:43]: Can we say with confidence that it _will_ work? not only _should_ :)

Comment (hakman) [reply cmd/kops/create_secret_dockerconfig.go:43]: Sure! 🤣

**Review by hakman — APPROVED** [review]

File: docs/cli/kops_create_secret_dockerconfig.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ Create a docker config.
 
 Create a new docker config, and store it in the state store. Used to configure docker on each master or node (i.e. for auth) Use update to modify it, this command will only create a new entry.
 
+ After creating a dockerconfig secret, a .docker/config.json file will be added to all newly created nodes without running kops update. This file should also apply auth to containerd, which is the default container runtime in kops 1.20 and above.
+
```
Comment (hakman) [inline docs/cli/kops_create_secret_dockerconfig.md:13]:
```suggestion
```

File: docs/cli/kops_create_secret_dockerconfig.md [hunk 3]
```diff
@@ -59,4 +59,3 @@ kops create secret dockerconfig [flags]
 ### SEE ALSO
 
 * [kops create secret](kops_create_secret.md)	 - Create a secret.
```
Comment (hakman) [inline docs/cli/kops_create_secret_dockerconfig.md:61]:
```suggestion
* [kops create secret](kops_create_secret.md)	 - Create a secret.

```

**Review by hakman — APPROVED** [review]

**Pull request merged**